### ISSUE TITLE
Sticky spinner below selections in the backend

### ISSUE THREAD

**Issue by trob** [joomlahenk/fabrik#371]
It's still showing the spinner after values are selected in
new list admin form/Data  'or Database table'
dbjoin element form/Data for Table, Value, Label

**Comment by joomlahenk** [comment]
Yes, this is from the beginning. I never found out how to solve this.

**Comment by skurvish** [comment]
Fixed in PR #374

**Comment by trob** [comment]
How did you find this?

I assume it must be also done in administrator\components\com_fabrik\models\fields\listfields.js to get rid of the dbjoin spinners.

**Comment by skurvish** [comment]
I will look also at the dbjoin spinners. The issue was caused by a previous PR that fixed a problem with modal dialogs not closing. But I found a way around it.

**Comment by skurvish** [comment]
I think PR #379 should fix the dbjoin as well.
